Cargo.toml

```diff
@@ -57,7 +58,10 @@ mock_base = [
     "env_logger",
 ]
 mock = ["mock_base"]
+test = ["mock"]
```

Comment:
does this mean use `mock` feature for test by default?
if so, maybe the CI script shall be updated as well?

Reply:
Yes this was a local change that I mistakenly put in, I will remove it though,